### PR TITLE
readme edits

### PR DESCRIPTION
**Pull request by MulverineX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-to ensure that even more
-- SSR is rendering as much HTML on the server as possible and sending
-ready to display HTML to the client instead of JS creating it client-side
-- Hydration is constructing vDOM from ready-to-display server-
-or statically rendered HTML
-- vDOM is a representation of DOM created by Vue to batch DOM changes for
-performance reason
+![frontend banner](https://cdn.discordapp.com/attachments/706901251560046654/727708398115750018/Frontend_Banner.png)
+
+## Modrinth's frontend service for SSR and hosting!
```
Comment (cfanoulis) [inline README.md:3]: no need for all the blabber
```suggestion
## Modrinth's frontend
```

**Comment by falseresync** [comment]
Yeah, that description is over the top

File: README.md (hunk 1 above)
Comment (falseresync) [reply README.md:3]: Kinda over the top

**Comment by falseresync** [comment]
Not actual anymore, gonna readd README later, also :concern: `cdn.discordapp.com`

**Pull request closed without merging**